plugins/webp-uploads/tests/test-picture-element.php

```diff
@@ -152,12 +134,19 @@ public function test_img_sizes_is_equal_to_picture_source_sizes_for_picture_elem
 			$this->markTestSkipped( "Mime type $mime_type is not supported." );
 		}
 
+		// Fallback to JPEG IMG.
+		update_option( 'perflab_generate_webp_and_jpeg', '1' );
+
+		// Create an image.
+		$attachment_id = self::factory()->attachment->create_upload_object( TESTS_PLUGIN_DIR . '/tests/data/images/leaves.jpg' );
```

Comment:
Joe, It will return only WEBP image.

```
<picture class="wp-picture-4" style="display: contents;">
  <source type="image/webp" srcset="http://localhost:8889/wp-content/uploads/2024/08/leaves-637-1024x683.webp 1024w, http://localhost:8889/wp-content/uploads/2024/08/leaves-637-300x200.webp 300w, http://localhost:8889/wp-content/uploads/2024/08/leaves-637-768x512.webp 768w" sizes="(max-width: 1024px) 100vw, 1024px">
  <img width="1024" height="683" src="http://localhost:8889/wp-content/uploads/2024/08/leaves-637-1024x683.webp" class="wp-image-4" alt="Green Leaves" decoding="async" loading="lazy" srcset="http://localhost:8889/wp-content/uploads/2024/08/leaves-637-1024x683.webp 1024w, http://localhost:8889/wp-content/uploads/2024/08/leaves-637-300x200.webp 300w, http://localhost:8889/wp-content/uploads/2024/08/leaves-637-768x512.webp 768w, http://localhost:8889/wp-content/uploads/2024/08/leaves-637-jpg.webp 1080w" sizes="(max-width: 1024px) 100vw, 1024px" />
</picture>
```

Reply:
I see. I think for this test class most of the tests are based on uploading a JPEG, and generating a WebP image with JPEG fallbacks. Instead of removing the fixture, We should probably set the `perflab_generate_webp_and_jpeg` option first before creating the image and then all tests that need to test a WebP with a JPEG fallback can use the same attachment.

```
// Fallback to JPEG IMG.
update_option( 'perflab_generate_webp_and_jpeg', '1' );
```

What do you think?

Reply:
Thanks. It works with the option. Updated in https://github.com/WordPress/performance/pull/1408/commits/0e07201f88482e95f979cfe7eb5884245ecb69cc